build.gradle

```diff
@@ -41,6 +41,7 @@ dependencies {
     
     modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
     modImplementation "me.lucko:fabric-permissions-api:${project.lucko_permissions_version}"
+    modImplementation "eu.pb4:sgui:${project.sgui_version}"
```

Comment:
Does this bundle sgui with this mod or is it an external dependency?

Reply:
This bundles SGui with the mod

Reply:
Okay great